localized/ja/UserGuide-chapter1.adoc

```diff
@@ -0,0 +1,149 @@
+= Chapter 1. OptaPlanner イントロダクション
+:awestruct-description: Chapter 1. OptaPlanner イントロダクション
+:awestruct-layout: localizedBase
+:awestruct-lang: jp
+:awestruct-priority: 1.0
+:showtitle:
+xxx
+
+////
+
+Failed, to be deleted soon >>>>>
```

Comment:
I presume a second PR is coming to delete these comments? I'll merge the current PR already.

Thank you for contributing this translation! Great work.
If you want to be recognized as a contributor, please submit a PR to add yourself in this file:
  https://github.com/droolsjbpm/optaplanner-website/blob/master/_config/users.yml